azure/datalake/store/transfer.py

```diff
@@ -387,7 +387,8 @@ def _update(self, future):
                     merge_future = self._submit(
                         self._merge, self._adlfs, dst,
                         [chunk for chunk, _ in sorted(cstates.objects,
-                                                      key=lambda obj: obj[1])])
+                                                      key=lambda obj: obj[1])], 
+                        overwrite=self._parent._overwrite)
```

Comment:
As long as those fields are all private to this module it's okay, but if whatever is stored in self._parent isn't then it's bad.